docs/data-analysis.md

```diff
@@ -1,30 +1,96 @@
-# Analyzing bot data
+# Analyzing bot data with Jupyter notebooks  
 
-You can analyze the results of backtests and trading history easily using Jupyter notebooks. A sample notebook is located at `user_data/notebooks/analysis_example.ipynb`. For usage instructions, see [jupyter.org](https://jupyter.org/documentation).
+You can analyze the results of backtests and trading history easily using Jupyter notebooks. Sample notebooks are located at `user_data/notebooks/`.  
 
-*Pro tip - Don't forget to start a jupyter notbook server from within your conda or venv environment or use [nb_conda_kernels](https://github.com/Anaconda-Platform/nb_conda_kernels)*
+## Pro tips  
 
-## Example snippets
+* See [jupyter.org](https://jupyter.org/documentation) for usage instructions.
+* Don't forget to start a jupyter notbook server from within your conda or venv environment or use [nb_conda_kernels](https://github.com/Anaconda-Platform/nb_conda_kernels)*
```

Comment:
> we should add a quickstart command people can run easily like jupyter notebook,
How would you suggest implementing? It could get complicated since the user may have a venv, conda environment, jupyter in a base environment, etc.

Reply:
> Don't forget to start a jupyter notbook server from within your conda or venv environment or use 

Assuming there's a novice jupyter user - knowing to run `jupyter notebook` | `jupyter lab`  from the activated environment will suffice to get that user going.

For more in-depth questions, refer to the jupyter documentation... 

If you have jupyter in the base environment and use nb_kernels you're an advanced user to me who does not need our quickstarter about this ... (it's about giving a quick-start - not providing jupyter documentation for everything)